### PR TITLE
MRKT-450 Provider returns an error that leads to infinite delivery loop

### PR DESCRIPTION
**Pull request by anastasiia-kryshtop**

### CODE REVIEW

**Review by vfisyuk-smartling — APPROVED** [review]

File: src/main/java/com/smartling/marketo/sdk/rest/transport/JaxRsHttpCommandExecutor.java [hunk 1]
```diff
@@ -51,6 +52,10 @@ public class JaxRsHttpCommandExecutor implements HttpCommandExecutor {
             "607",  // Daily quota
             "615"   // Concurrent request limit
     );
+    private static final Set<String> UPDATE_CONTENT_NOT_ALLOWED_CODES = ImmutableSet.of("702");
+    private static final String UPDATE_CONTENT_NOT_ALLOWED_MESSAGE_START = "Update content";
```
Comment (sl-pkryshtop) [inline src/main/java/com/smartling/marketo/sdk/rest/transport/JaxRsHttpCommandExecutor.java:56]: maybe use pattern match `^Update content.*not allowed.$`

File: src/main/java/com/smartling/marketo/sdk/rest/transport/JaxRsHttpCommandExecutor.java [hunk 2]
```diff
@@ -51,6 +52,10 @@ public class JaxRsHttpCommandExecutor implements HttpCommandExecutor {
...
+    private static final String UPDATE_CONTENT_NOT_ALLOWED_MESSAGE_END = "not allowed.";
+
     private static final Set<String> NOT_FOUND_CODES = ImmutableSet.of("702", "710");
```
Comment (sl-pkryshtop) [inline src/main/java/com/smartling/marketo/sdk/rest/transport/JaxRsHttpCommandExecutor.java:59]: looks like, NOT_FOUND 702 code is now treated as NOT_ALLOWED, maybe remove it from here?

**Review by sl-pkryshtop — APPROVED** [review]

File: src/main/java/com/smartling/marketo/sdk/rest/transport/JaxRsHttpCommandExecutor.java (hunk 2 above)
Comment (anastasiia-kryshtop) [reply src/main/java/com/smartling/marketo/sdk/rest/transport/JaxRsHttpCommandExecutor.java:59]: I'm not sure, that 702 code should be removed from NOT_FOUND. In the Marketo documentation, it describes as ```702	No data found for given search scenario	No records matched the given search parameters.```

Comment (sl-pkryshtop) [reply src/main/java/com/smartling/marketo/sdk/rest/transport/JaxRsHttpCommandExecutor.java:58]: my mistake

**Pull request merged**